CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -231,16 +258,37 @@ void SiPixelLorentzAnglePCLHarvester::dqmEndJob(DQMStore::IBooker& iBooker, DQMS
     min_drift_ = hists.h_drift_depth_adc_[1]->getAxisMin(1);
     max_drift_ = hists.h_drift_depth_adc_[1]->getAxisMax(1);
   } else {
-    hist_drift_ = 200;
+    hist_drift_ = 100;
     hist_depth_ = 50;
-    min_drift_ = -1000.;
-    max_drift_ = 1000.;
+    min_drift_ = -500.;
+    max_drift_ = 500.;
   }
 
-  iBooker.setCurrentFolder("AlCaReco/SiPixelLorentzAngleHarvesting/");
+  iBooker.setCurrentFolder(fmt::format("{}Harvesting", dqmDir_));
   MonitorElement* h_drift_depth_adc_slice_ =
       iBooker.book1D("h_drift_depth_adc_slice", "slice of adc histogram", hist_drift_, min_drift_, max_drift_);
 
+  // retrieve the number of bins from the other monitoring histogram
+  const auto& maxSect = hists.h_bySectOccupancy_->getNbinsX();
+
+  // this will be booked in the Harvesting folder
+  iBooker.setCurrentFolder(fmt::format("{}Harvesting/SectorMonitoring", dqmDir_));
+  MonitorElement* h_bySectLA_ =
+      iBooker.book1D("h_bySectorLA",
+                     "tan#theta_{LA}/B by sector ;pixel sector;measured tan(#theta_{LA})/B [1/T]",
+                     maxSect,
+                     -0.5,
+                     maxSect + 0.5);
+  MonitorElement* h_bySectChi2_ = iBooker.book1D(
+      "h_bySectorChi2", "#chi^{2}/ndf by sector;pixel sector; fit #chi^{2}/ndf", maxSect, -0.5, maxSect + 0.5);
+
+  // copy the bin labels from the occupancy histogram
+  for (int bin = 1; bin < maxSect; bin++) {
+    const auto& binName = hists.h_bySectOccupancy_->getTH1()->GetXaxis()->GetBinLabel(bin);
```

Comment:
is this tolerated / thread safe? @cms-sw/dqm-l2 
I didn't find a `MonitorElement::getBinLabel()` method at https://github.com/cms-sw/cmssw/blob/master/DQMServices/Core/interface/MonitorElement.h